phpBB/phpbb/db/migration/data/v32x/enable_plupload_config.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace phpbb\db\migration\data\v32x;
+
+class enable_plupload_config extends \phpbb\db\migration\migration
```

Comment:
Please rename this file to `add_plupload_config.php` as this IMHO better describes the purpose of this migration.

Reply:
Thank you @marc1706 for your review and feedback.  I have made all these changes. Would you like me to create another migration file for 3.3.0 > 3.3.1 (or whichever version this makes it into)? Or will the file be replicated and amended for correct version number when merged into 3.3.x branch?

Reply:
The changes will be merged upstream into 3.3 and master so there is no need for that.